### ISSUE TITLE
Goes into infinite spin on "verify-mail" - freezes all www.nexusmods.com tabs

### ISSUE THREAD

**Issue by zb-z** [Nexus-Mods/web-issues#1067]
## Steps to reproduce:

URL: https://users.nexusmods.com/register 
- click [VERIFY EMAIL]

Goes into infinite spin - most probably some JS code. Chrome (latest, uBlock turned off for domain) shows its spinner that normally happens only if a page takes long time to load and it does "think" that page hasn't been loaded - DOM is de-facto empty (one empty head and body node). Eventually Chrome pops up "page non responsive"  for ALL nexusmods.com tabs. I had 3 { https://www.nexusmods.com/civilisationvi/mods/categories/2/, https://www.nexusmods.com/civilisationvi/mods/89?tab=files, https://users.nexusmods.com/register  } 

Edge reports a JS taking up too much resources much sooner. 

Sometimes it does create partial UI [4 white fields on back background], sometimes just a blank, with black background. Nodes for these elements are not visible in Dev Console (Chrome, haven't tried edge).

Server did send codes.

## Expected result:
                                - to actually work :-))

## Actual result:
                             - read above

## Comments:

All pages reported as non responsive can mean unholy entanglement  - pages shouldn't "know" about each other or try to communicate in any shape or form. Or some stupid "reactive" JS "calling home" all the time and receiving something bad from server when one page gets stuck?

4 input fields fields for 4 letters ??? That's asking got trouble on it's own - keep it simple stupid. If it was one simple text field and a normal page (no JS running just in order to "look impressive" or no rational reason) it would have worked no matter what and if anything would ever get stuck I'd have a decent chance of just entering the data via Dev Console. 

Many sites send URL in e-mail with encrypted credentials and verification is simple click. Granted, that leaves some stray pages/tebs/windows to be closed but is much more stable and efficient (you don't have to retype anything).

Such sites also do basic registration first => verification is transactional - the second you click account is active.  Yes, that , means that servers have to do weekly garbage collection (delete non verified accounts) and keep a flag so that a guy can go back, log in, and then click to send verification e-mail again.

**Comment by Pickysaurus** [comment]
We have not been able to reproduce this issue. It may be related to your browser settings or addons.

**Comment by flipwise** [comment]
I'm having the same freeze when trying to verify my email. The site makes Chromium eat up the entire CPU core. I had to do the registration in Firefox.

OS: Manjaro Linux
Browser: Chromium Version 79.0.3945.88 (Official Build) Arch Linux (64-bit)

Addons don't seem to be responsible as I have the same ones in the two browsers.

**Comment by grey-panther** [comment]
Hi! I have the same problem with the "verify email" page. So I couldn't register on nexusmods.com using my Chrome browser (I tried to do this even without browser addons/extensions - in Incognito mode).
Windows 10 x64 (10.0 build 17763)
Google Chrome Version 79.0.3945.117 (Official Build) (64-bit)

Also, I tried to register using another chrome-based browser (Yandex Browser) and got the same problem there.

Finally, I managed to register using Microsoft Edge...
